x-pack/plugins/profiling/public/components/frame_information_window/get_impact_rows.tsx

```diff
@@ -25,6 +25,7 @@ export function getImpactRows({
   countExclusive: number;
   totalSamples: number;
   totalSeconds: number;
+  isApproximate: boolean;
```

Comment:
Looks like you declared the `isApproximate` here but you're not using it. Can it be removed?